inference-engine/src/snippets/CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ ie_developer_export_targets(${TARGET_NAME})
 
 # install
 
-# TODO: uncomment once snippets are integrated into CPU plugin
-# install(TARGETS ${TARGET_NAME}
-#         RUNTIME DESTINATION ${IE_CPACK_RUNTIME_PATH} COMPONENT core
-#         LIBRARY DESTINATION ${IE_CPACK_LIBRARY_PATH} COMPONENT core)
+install(TARGETS ${TARGET_NAME}
+        RUNTIME DESTINATION ${IE_CPACK_RUNTIME_PATH} COMPONENT core
+        ARCHIVE DESTINATION ${IE_CPACK_ARCHIVE_PATH} COMPONENT core
```

Comment:
```
ARCHIVE DESTINATION ${IE_CPACK_ARCHIVE_PATH} COMPONENT core
```
Must be removed. They are not needed on Windows because end users don't link against this library